tezos/timing/src/schema_stats.sql

```diff
@@ -9,7 +9,9 @@ CREATE TABLE IF NOT EXISTS blocks (
   checkout_time_irmin REAL,
   checkout_time_tezedge REAL,
   commit_time_irmin REAL,
-  commit_time_tezedge REAL
+  commit_time_tezedge REAL,
+  start_time INTEGER DEFAULT CURRENT_TIMESTAMP NOT NULL,
```

Comment:
Not sure about this default here, I think it will store the value as a string formatted date and time (sqlite doesn't care much about what the type of the column is). I would rather set it as `REAL` without default and pass the time directly from Rust as a floating point number of the current time (we want better resolution than just 1 second).

Reply:
Another issue that I am not thinking about is that if the recording thread lags behind this value is going to be incorrect. I think the best thing to do is to get the timestamp inside the hook function, and send it as part of the message sent to the recording thread.